config/default.js

```diff
@@ -0,0 +1,79 @@
+// CONFIG defaults (aka PRODUCTION)
+// WARNING: No test/stage/dev/development config should
+// live here.
+
+/* eslint-disable object-shorthand */
```

Comment:
I'll need to check - karma pulls this in and that's pure node - not sure if that's using babel.